### PR TITLE
Enable tendrl-gluster-integration service during import

### PR DESCRIPTION
**Pull request by nnDarshan**
Currently tendrl-gluster-integration service is not
enabled during import. Because of this it doesnt start
automatically on reboot. This patch adds code to enable
the service during import.

tendrl-bug-id: Tendrl/gluster-integration#294
Signed-off-by: Darshan N <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@nthomas-redhat @shtripat @r0h4n Please review

**Comment by r0h4n** [comment]
@nnDarshan thanks. What other Tendrl services need such fix?

**Comment by nnDarshan** [comment]
> @nnDarshan thanks. What other Tendrl services need such fix?

Others would not need any such fix as those are started by tendrl-ansible and its ansible's responsibility to enable the service. AFAIK this is the only service we are starting from code

File: tendrl/commons/flows/import_cluster/gluster_help.py [hunk 1]
```diff
@@ -146,7 +146,12 @@ def import_gluster(parameters):
         )
     )
     os.chmod(_gluster_integration_conf_file_path, 0o640)
+
+    if NS.config.data['package_source_type'] == 'rpm':
+        command = "systemctl enable tendrl-gluster-integration"
+        subprocess.Popen(command.split())
```
Comment (r0h4n) [inline tendrl/commons/flows/import_cluster/gluster_help.py]: please handle case where command fails and log appropriate error

Comment (nnDarshan) [reply tendrl/commons/flows/import_cluster/gluster_help.py]: ack

**Pull request merged**